### PR TITLE
navbar layout

### PR DESCRIPTION
**Pull request by CUBOFIG**
# Description
Creation of the design of the navigation bar and installation of the yarn instance that will allow us to perform the Smooth Scroll.

 ## Screenshots
imagen
imagen

### CODE REVIEW

**Review by JuanequeX — DISMISSED** [review]
LGTM

File: app/views/home/partials/_nav.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    
+    <div>
+      <%= link_to "Login", new_user_session_path %>
+      <%= link_to "Register", new_user_registration_path %>
+    </div>
+
+    <div>
+      <%# SVG %>
+    </div>
+  </div>
```
Comment (YilKanda) [inline app/views/home/partials/_nav.html.erb:27]: Question, why do you need to group the elements in a div, if you are grouping them in a section?

Comment (CUBOFIG) [reply app/views/home/partials/_nav.html.erb:27]: We are using SASS for the development of styles, so mainly when I work with sass I add a main class to wrap the entire element and the second div would be to make the container where the use of flexbox would be reflected.

example: 

image

**Review by YilKanda — DISMISSED** [review]
LGTM!

File: app/views/home/partials/_nav.html.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <ul>
+      <li>
+        <a href="#products" data-controller="scroll-to">PRODUCTS</a>
+      </li>
+      <li>
+        <a href="#pricing" data-controller="scroll-to">PRICING</a>
+      </li>
+      <li>
+        <a href="about-us" data-controller="scroll-to">ABOUT US</a>
+      </li>
```
Comment (fabiannabi) [inline app/views/home/partials/_nav.html.erb:16]: In this section you can use as well the rails helper for `link_to` just like you did below. To keep it consistent i would suggest using them on both cases, what do you think?

**Review by fabiannabi — CHANGES_REQUESTED** [review]

File: app/views/home/partials/_nav.html.erb (hunk 2 above)
Comment (CUBOFIG) [reply app/views/home/partials/_nav.html.erb]: Thanks, that solution was better than the one we are implementing.

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by EdsonMorfin — APPROVED** [review]
LGTM

**Pull request merged**